engine/src/main/java/org/terasology/rendering/dag/Node.java

```diff
@@ -15,7 +15,6 @@
  */
 package org.terasology.rendering.dag;
 
-
 public interface Node {
 
     void initialise();
```

Comment:
This is one area where we may actually have no standard, I'm pretty sure we have both variants scattered through the code base, depending on who was writing it ;-)